### PR TITLE
Update ie-to-microsoft-edge-redirection.md

### PR DESCRIPTION
**Pull request by HarneetSidhana**
Adding redirection experience from https://github.com/MicrosoftDocs/Edge-Enterprise/tree/public/edgeenterprise. I was unable to create new files to upload the screenshots. The screenshots that need to be updated are located here --> https://github.com/MicrosoftDocs/Edge-Enterprise/tree/public/edgeenterprise/media. @jm-trd-ms can you please review the changes, upload the files and make sure it works.

### CODE REVIEW

File: microsoft-edge/web-platform/ie-to-microsoft-edge-redirection.md [hunk 1]
```diff
@@ -26,26 +26,53 @@ The IE compatibility List only adds a website when the following actions occur.
...
+*   Gives the user the option to explore Microsoft Edge.    
+    
+When a website is redirected from Internet Explorer to Microsoft Edge, one of the following actions occurs.
 
-When a site is redirected from Internet Explorer to Microsoft Edge, the Internet Explorer tab that started loading the site is closed if it had no prior content. Otherwise, the active tab view goes to a  Microsoft [support](https://support.microsoft.com/office/the-website-you-were-trying-to-reach-doesn-t-work-with-internet-explorer-8f5fc675-cd47-414c-9535-12821ddfc554?ui=en-US&rs=en-US&ad=US) page that explains why the site was redirected to Microsoft Edge.
+*   If the active IE tab had no prior content, it is closed.  
+*   If the active IE tab had prior content, it navigates to the [Microsoft support page that explains why the website was redirected to Microsoft Edge][MicrosoftSupportOfficeTheWebsiteYouWereTryingToReachDoesntWorkWithInternetExplorer].  
 
 > [!NOTE]
 > After
```
Comment (HarneetSidhana) [inline microsoft-edge/web-platform/ie-to-microsoft-edge-redirection.md:74]: This is missing the following sentence. Can you please update 

"After a redirection users can go back to using Internet Explorer for sites that are not on the Internet Explorer incompatibility list."

Comment (jm-trd-ms) [reply microsoft-edge/web-platform/ie-to-microsoft-edge-redirection.md]: Updated.

**Comment by HarneetSidhana** [comment]
Thanks LGTM. Lemme know once it is merged with master.

**Comment by jm-trd-ms** [comment]
#sign-off offline and merged.

**Pull request merged**